### PR TITLE
Initial commit for Moodle 3.1

### PR DESCRIPTION
**Pull request by mgifford**
There are validation issues that I don't understand here.
Invalid: data/product/version must be string, data/notes must be string, data/chapters/success_criteria_level_a/criteria/4/components/0/adherence/notes must be string, data/chapters/success_criteria_level_a/criteria/5/components/0/adherence/notes must be string, data/chapters/success_criteria_level_aaa/notes must be string, data/chapters/success_criteria_level_aaa/criteria/7/components/0/adherence/notes must be string, data/chapters/success_criteria_level_aaa/criteria/12/components/0/adherence/notes must be string

I can fill in the notes.

### CODE REVIEW

File: opat/Moodle-3.yaml [hunk 1]
```diff
@@ -0,0 +1,807 @@
...
+  name: Moodle
+  version: 3.1
+  description: The world's most customisable and trusted open-source learning management system.
+author:
+  name: Sander Bangma & Jun Pataleta
+  company_name: Moodle Pty Ltd
+  address: PO Box 303, West Perth WA 6872, Australia
+  email: support@moodle.com
+  website: https://docs.moodle.org
+notes:
```
Comment (dmundra) [inline opat/Moodle-3.yaml:12]: The validator complains when this is provided with no data. If you do `notes: ""` that is valid or alternatively remove the reference altogether.

Comment (dmundra) [reply opat/Moodle-3.yaml:12]: @mgifford I think all the current validation issues are the same, empty notes. So the fix above will work for all of them.

File: opat/Moodle-3.yaml [hunk 2]
```diff
@@ -0,0 +1,807 @@
+title: Moodle Accessibility Conformance Report
+product:
+  name: Moodle
+  version: 3.1
```
Comment (dmundra) [inline opat/Moodle-3.yaml:4]: @mgifford version needs to be a string as well, `version: "3.1"`

**Review by mgifford — COMMENTED** [review]
I also wanted to note that Moodle did the WCAG version which includes WCAG 2.1 which we aren't evaluating yet.  I just left out those cells.

**Comment by dmundra** [comment]
@mgifford did you want to generate the markdown and HTML versions as well?

**Comment by mgifford** [comment]
Added. Ok to commit now @dmundra ?

**Review by dmundra — APPROVED** [review]
Looking good.

**Pull request merged**